Docs: Improve Hacktoberfest Guidelines (#2324)
EddieHubCommunity/LinkFree: README.md

> ### A note for Hacktoberfest participants:
>
> - Pull requests which add or edit your information in a `data/${yourname}.json` file will NOT be counted for Hacktoberfest.

> > - Pull requests which improve the codebase, documentation, or other aspects of the project and are in line with the core values of the event will count. The project contains the `hacktoberfest` topic, so the PRs will be automatically counted towards Hacktoberfest. However, profile update PRs that have the label `invalid` will not be counted towards this event.

Reviewer: How does this sound

```suggestion
> - Pull requests which improve the codebase, documentation, or other aspects of the project and are in line with the core values of the event will count. The project contains the `hacktoberfest` topic, so the PRs will be automatically counted towards Hacktoberfest. However, profile update PRs will have the label `invalid` and therefore won't be counted towards this event.
```
